internal/integration/testdata/mysql/cli-migrate-diff.txt

```diff
@@ -0,0 +1,76 @@
+exec mkdir migrations
+
+! atlas migrate diff --to file://1.hcl --dir file://migrations
+stderr '"dev-url" not set'
+
+! atlas migrate diff --dev-url mysql://devdb --dir file://migrations
+stderr '"to" not set'
+
+atlas migrate diff --dev-url URL --to file://./1.hcl first
+cmpmig 0 1.sql
+
+atlas migrate diff --dev-url URL --to file://./2.hcl second
+cmpmig 1 2.sql
+
+-- 1.hcl --
+schema "script_cli_migrate_diff" {}
+
+table "users" {
+  schema = schema.script_cli_migrate_diff
+  column "id" {
+    null = false
+    type = bigint
+    auto_increment = true
+  }
+  primary_key  {
+    columns = [column.id]
+  }
```

Comment:
You can configure the `charset` and `collate` (e.g. `latin1` `latin1_swedish_ci`) to avoid the duplicate SQL files below.